task.sql

```diff
@@ -1,11 +1,15 @@
--- Use our database
-USE ShopDB; 
+USE ShopDB;
 
--- Some data should be created outside the transaction (here)
+INSERT INTO Orders (CustomerID, Date)
+VALUES (1, "2023-01-01");
 
--- Start the transaction 
 START TRANSACTION; 
 
--- And some data should be created inside the transaction 
+INSERT INTO OrderItems (ProductID, Count, OrderID)
+VALUES (1, 1, 1);
 
-COMMIT; 
\ No newline at end of file
+UPDATE Products
+SET WarehouseAmount = WarehouseAmount - 1
+WHERE ID = '1';
+
+COMMIT;
```

Comment:
![image](https://github.com/user-attachments/assets/ae639b4e-d85c-4e4d-9253-b04682614df8)
Add Eof(End-of-file), need to add blank line at the end